polaris/experimental/_dataset_v2.py

```diff
@@ -40,6 +48,8 @@ class DatasetV2(BaseDataset):
     """
 
     version: ClassVar[Literal[2]] = 2
+    _zarr_md5sum_manifest: str | None = PrivateAttr(None)
```

Comment:
**Suggestion:** I don't think having this property here adds value, because it effectively seems to duplicate the existing `_md5sum` property (coming from the `ChecksumMixin`). 

I also think the name is confusing because how it's used here (i.e. it stores the hash of the manifest) is different from how it's defined in the `BaseDataset` class (i.e. it stores the whole manifest).

Reply:
Yeah I was thinking this too during implementation. Agreed, let's get rid of it.